<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelData">
           <property name="text">
            <string>Data:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditData"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBoxCalculateModel"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonStatistics">
             <property name="text">
              <string>Calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelConsole">
           <property name="text">
            <string>Console:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditInput"/>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditConsole">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelResult">
          <property name="text">
           <string>Result:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditResult">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionFusion"/>
    <addaction name="actionWindows"/>
    <addaction name="actionWindowsVista"/>
    <addaction name="separator"/>
    <addaction name="actionQdarkstyle"/>
    <addaction name="actionQdarkgraystyle"/>
    <addaction name="actionMyStyle"/>
    <addaction name="actionResetStyle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStyle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionQdarkstyle">
   <property name="text">
    <string>Qdarkstyle</string>
   </property>
  </action>
  <action name="actionQdarkgraystyle">
   <property name="text">
    <string>Qdarkgraystyle</string>
   </property>
  </action>
  <action name="actionMyStyle">
   <property name="text">
    <string>MyStyle</string>
   </property>
  </action>
  <action name="actionResetStyle">
   <property name="text">
    <string>ResetStyle</string>
   </property>
  </action>
  <action name="actionWindowsVista">
   <property name="text">
    <string>WindowsVista</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFusion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQdarkgraystyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQdarkstyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMyStyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindows</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindowsVista</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetStyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditInput</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>line_edit_return_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStatistics</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxCalculateModel</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>combobox_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_clicked()</slot>
  <slot>action_triggered()</slot>
  <slot>line_edit_return_pressed()</slot>
  <slot>combobox_activated()</slot>
 </slots>
</ui>
